% Template for PLoS
% Version 1.0 January 2009
%
% To compile to pdf, run:
% latex plos.template
% bibtex plos.template
% latex plos.template
% latex plos.template
% dvipdf plos.template
\documentclass[11pt]{article}
%\usepackage{natbib}
\usepackage{hyperref}
\usepackage{lineno}

% amsmath package, useful for mathematical formulas
\usepackage{amsmath}
% amssymb package, useful for mathematical symbols
\usepackage{amssymb}

% graphicx package, useful for including eps and pdf graphics
% include graphics with the command \includegraphics
\usepackage{graphicx}

% cite package, to clean up citations in the main text. Do not remove.
\usepackage{cite}

\usepackage{color} 

% Use doublespacing - comment out for single spacing
%\usepackage{setspace} 
%\doublespacing


% Text layout
\topmargin 0.0cm
\oddsidemargin 0.5cm
\evensidemargin 0.5cm
\textwidth 16cm 
\textheight 21cm
\linespread{1.2}
% Bold the 'Figure #' in the caption and separate it with a period
% Captions will be left justified
\usepackage[labelfont=bf,labelsep=period,justification=raggedright]{caption}

% Use the PLoS provided bibtex style
\bibliographystyle{plos2009}

% Remove brackets from numbering in List of References
\makeatletter
\renewcommand{\@biblabel}[1]{\quad#1.}
\makeatother


% Leave date blank
\date{}

\pagestyle{myheadings}
%% ** EDIT HERE **


%% ** EDIT HERE **
%% PLEASE INCLUDE ALL MACROS BELOW

%% END MACROS SECTION

\begin{document}

% Title must be 150 characters or less
\begin{flushleft}
{\Large
\textbf{Characterization of the transcriptome, nucleotide sequence polymorphism, and natural selection in the desert adapted mouse \textit{Peromyscus eremicus}}
}
% Insert Author names, affiliations and corresponding author email.
\\
Matthew D. MacManes$^{1}$, 
Michael B. Eisen $^{2}$, 
\\
\bf{1} Department of Molecular, Cellular and Biomedical Sciences, University of New Hampshire, Durham NH, USA
\\
\bf{2} HHMI and University of California, Berkeley, Berkeley, CA, USA
\\
$\ast$ E-mail: macmanes@gmail.com, @PeroMHC
\end{flushleft}

\linenumbers

% Please keep the abstract between 250 and 300 words
\section*{Abstract}



\section*{Introduction}

For biologists interested in understanding the relationship between fitness, genotype, and phenotype, modern sequencing technologies provide for an unprecedented opportunity to gain a deep understanding of genome level processes that together, underlie adaptation. Unlike more traditional approaches (e.g. QTL mapping), second generation sequencing allow researchers to access genome wide patterns of gene expression, nucleotide variation and ulitimately pattens of natural selection in non-model organisms lacking an extensive set of genomic tools. This approach has been used widely in recent years in characterizing X, Y and Z. \\

One interesting example of adaptation lies in animal's ability to survive desert conditions. Here, heat \emph{and} drought provide for powerful selective forces, testing animals' ability to osmoregulate and thus to survive \cite{Walsberg:2000uu}. Indeed, the maintenance of water balance in animals is one of the most important physiologic processes, and is critical to desert survival. Mammals are exquisitely sensitive to changes in osmolality, with slight derangement eliciting physiologic compromise.  When the loss of water exceeds dietary intake, dehydration - and in extreme cases, death - can occur.  Unlike most mammals, animals living in desert habitats are subjected to long periods of extreme heat and intense drought.  As a result, desert animals have evolved mechanisms through which physiologic homeostasis is maintained despite severe and prolonged dehydration. Understanding these mechanismsm will significantly enhance our understanding of the physiologic processes underlying osmoregulation in extreme environments, having implications for studies of human health, conservation, and climate change. \\

Genes responsible for the maintinance of water balance are well characterized in model organisms such as mice \cite{Tatum:2009kp}, rats \cite{Romero:2007fs,Rojek:2006dg,NIELSEN:1995uq}, and humans \cite{Mobasheri:2007tt,Bedford:2003wa,NIELSEN:1999vg}. In addition to these studies, a ong standing interest in deserts adaptation has resulted in a number of studies that looked at the morphology or expression of single genes in the renal tissues of desert adapted rodents \textit{Phyllotis darwini} \cite{Gallardo:2005fm}, \textit{Psammomys obesus} \cite{Kaissling:1975uu}, and \textit{Perognathus penicillatus} \cite{Altschuler:1979du}. More recently, full renal transcriptomes have been generated for \textit{Dipodomys spectabilis} and \textit{Chaetodipus baileyi} \cite{Marra:2014de} as well as \textit{Abrothrix olivacea} \cite{Giorello:2014iv}. \\

These studies provide a rich context for the current and future work, aimed at developing a synthetic understanding of the the genetic and genomic underpinnings of desert adaptation in rodents. As a first step, we have sequenced, assembled, and characterized the transcriptome (using four tissue types - liver, kidney, testes, brain), of a desert adapted cricetid rodent endemic to the Southwest United States \cite{Veal:2001vp}, \textit{Peromyscus eremicus}. These animals have a lifespan typical of small mammals, and therefore an individual may live it's entire life without ever drinking water. These rodents have distinct advantage over other desert animals (e.g. \textit{Dipodomys}) in that they breed readily in captivity, which enables future laboratory studies of the phenotype of interest. In addition the focal species is positioned in a clade of well known animals (e.g. \textit{P. californicus}, \textit{P. maniculatus} and \textit{P. polionotus}) \cite{Bradley:2007jo} with growing genetic and genomic resources \cite{Shorter:2014kw,Panhuis:2011co,Shorter:2012ib} which together suggest that future comparative studies are possible. \\


While the elucidation of the mechanisms underlying adaptation to desert survival is beyond the scope of this manuscript, we aim here to lay the groundwork by characterizing the transcriptome from four distinct tissues (brain, liver, kidney, testes). These data will be included in current larger effort aimed at sequencing the entire genome. Further, via sequencing the renal tissue of a total of 15 additional animals, we characterize nucleotide polymorphism and genome wide patterns of natural selection. Together, these investigations will aid in our overarching goal -- to understand the genetic bases of adaptation in \textit{P. eremicus}.

\section*{Materials and Methods}

\subsection*{Animal Collection and Study Design}

To begin to understand how genes may underlie desert adaptation, I collected ~15 individuals from a single population \textit{P. eremicus} over a two year time period (2012-2013). These individuals were captured in live traps, then euthanized using isoflurane overdose and decapitation. Immediately post-mortem, the abdominal and pelvic organs were removed, cut in half (in the case of kidney), placed in RNAlater and flash frozen in liquid Nitrogen. Removal of the brain, with similar preservation techniques, followed that. Time from euthanasia to removal of all organs never exceeded five minutes. Samples were transferred to a -80C freezer at a later date. These procedures were approved by the University of California Berkeley Animal Care and Use Committee and follow guidelines established by the American Society of Mammalogy for the use of wild animals in research \cite{Sikes:2011dz}.  

\subsection*{RNA extraction and Sequencing}

Total RNA was extracted from each tissue using a TRIzol extraction (Invitrogen) following the manufacturers instructions. Because preparation of an RNA library suitable for sequencing is dependent on having high
quality, intact RNA, a small aliquot of each total RNA extract was
analyzed on a Bioanalyzer 2100 (Agilent, Palo Alto, CA, USA).
Following confirmation of sample quality, the reference sequencing libraries
were made using the TruSeq stranded RNA prep kit
(Illumina), while an unstranded TruSeq kit was used to construct the other sequencing libraries. A unique index was ligated to each sample to allow for multiplexed sequencing. Reference libraries (n=4 tissue types) were then pooled to
contain equimolar quantities of each individual library and
submitted for Illumina sequencing using two lanes of 150nt paired end sequencing using the rapid-mode of the HiSeq 2500 sequencer at The Hubbard Center for Genome Sciences (University of New Hampshire). The remaining 14 libraries were similarly multiplexed and sequenced in a mixture of 100nt paired and single end across two lanes of an Illumina HiSeq 2000 at the Vincent G. Coates Gnome Center (University of California, Berkeley).

\subsection*{Sequence Data Preprocessing and Assembly}

Following recommendations from MacManes \cite{MacManes:2014io} and Mbandi \cite{Christoffels:2014bg}, adapter sequence contamination was removed, and low quality nucleotides (defined as \textsc{Phred} \textless 2) were removed from the dataset using the program Trimmomatic version 0.32  \cite{Lohse:2012fg}. We concatenated sequence data from each reference tissue type and assembled them jointly using the Trinity beta version released 16 March 2014 \cite{Haas:2013jq}. We used flags indicating the stranded nature of sequencing reads and set maximum allowable physical distance between read pairs to 999nt. The assembly was conducted on the \textsc{Xsede} computer resource Blacklight. To filter the raw sequence assembly, I estimated TPM for each assembled sequence using bwa-mem version 0.75 \cite{Li:2013wn} and eXpress version 1.51 \cite{Roberts:2012dh}, removing all contigs whose expression was less than TPM=1 \cite{Haas:2013jq}. \\

\subsection*{Assembled Sequence Annotation}

From the filtered assembly, I extracted putative coding sequences using Transdecoder version 16Jan2014 (\url{http://transdecoder.sourceforge.net/}). These putative protein coding sequences were annotated using default settings of the blastx algorithm \cite{Camacho:2009fc} against the SwissProt database downloaded on 1 March 2014. Because transcriptome assemblies typically contain non-coding elements (e.g. ncRNA) in addition to protein coding sequence, we annotated the entire filtered dataset using the NCBI nt dataset, downloaded on 1 March 2014. Lastly, because rapidly evolving nucleotide sequences may evade detection by blast algorithms, we used HMMER3 \cite{Wheeler:2013gj} to search for conserved protein domains contained in the Pfam database \cite{Punta:2012ko}. \\

To identify sequences unique to each tissue type, I mapped sequence reads from each tissue type to the reference assembly using bwa-mem. We estimated expression individually for the four tissues. Interesting patterns of expression, including instances where expression was limited to a single tissue type were identified. \\ 

\subsection*{Population Genomics}

In addition to the reference individual sequenced at four different tissue types, we sequenced 15 other conspecific individuals from the same population, located in Palm Desert, California. Sequence data were mapped to the reference transcriptome using bwa-mem. The alignments were sorted and converted to BAM format, then passed to the program \textsc{Angsd}, which was used for calculating the folded site frequency spectrum (SFS) and Tajima's D \cite{Korneliussen:2013uz}. \\


% Results and Discussion can be combined.
\section*{Results}

\subsection*{RNA extraction, Sequencing, Assembly, Mapping}

RNA was extracted from the hypothalamus, renal medulla, testes, or liver from each individual using sterile technique. TRIzol extraction resulted in a large amount of high quality (RIN $\geq$ 8) total RNA, which was used as input. Libraries were constructed as per the standard Illumina protocol, and ere sequenced as described above. The number of reads per library varied from 56 million strand-specific paired-end reads in Peer360 kidney, to 9 million single-end reads in Peer321. Adapter sequence contamination and low-quality nucleotides were eliminated, which resulted in a loss of \textless 2\% of reads. \\

Transcriptome assembly was accomplished using the program Trinity. The raw assembly contained 743314 assembled sequenced measuring 418Mb. This assembly was filtered using TMP \textgreater 1 as a threshold. The filtered assembly contained 130764 sequences measuring 149Mb. from this filteres dataset, I extracted 64355 putative coding sequences (60Mb). Of these 64355 sequences, 37960 were complete exons (containing both start and stop codons), while other were either truncated at the 5-prime end (16880 sequences), 3-prime end (4203 sequences), or were internal (5312 sequencing having neither stop nor start codon). 



\subsection*{Subsection 2}

\section*{Discussion}

% You may title this section "Methods" or "Models". 
% "Models" is not a valid title for PLoS ONE authors. However, PLoS ONE
% authors may use "Analysis" 

% Do NOT remove this, even if you are not including acknowledgments
\section*{Acknowledgments}


%\section*{References}
% The bibtex filename
\bibliography{formatted1.bib}

\section*{Figure Legends}
%\begin{figure}[!ht]
%\begin{center}
%%\includegraphics[width=4in]{figure_name.2.eps}
%\end{center}
%\caption{
%{\bf Bold the first sentence.}  Rest of figure 2  caption.  Caption 
%should be left justified, as specified by the options to the caption 
%package.
%}
%\label{Figure_label}
%\end{figure}


\section*{Tables}

\vspace{10mm}
\textbf{\hypertarget{Table 1}{Table 1}} \\
\begin{center}
\begin{tabular}{l|c }
\textsc{Dataset}& \textsc{Num. Raw Reads} \\
\hline 
\textsc{Peer360 Testes} & 32M PE  \\
\textsc{Peer360 Liver} & 53M PE   \\
\textsc{Peer360 Kidney} & 56M PE   \\
\textsc{Peer360 Brain}  & 23M PE \\
\textsc{Peer305}  & 19M PE  \\
\textsc{Peer308}  & 15M PE  \\
\textsc{Peer319}  & 14M PE  \\
\textsc{Peer321}  & 9M SE  \\
\textsc{Peer340}  & 16M PE  \\
\textsc{Peer352}  & 14M PE  \\
\textsc{Peer354}  & 9M SE  \\
\textsc{Peer359}  & 14M PE  \\
\textsc{Peer365}  & 16M PE \\
\textsc{Peer366}  & 16M PE  \\
\textsc{Peer368}  & 14M PE  \\
\textsc{Peer369}  & 14M PE  \\
\textsc{Peer372}  & 17M SE  \\
\textsc{Peer373}  & 23M SE  \\
\textsc{Peer380}  & 16M SE  \\
\textsc{Peer382}  & 14M SE  \\
\end{tabular}
\begin{quote}
\small{Table 1. The number of sequencing reads per sample}
\end{quote}


\end{center}  


\vspace{5mm}


\end{document}

