% Template for PLoS
% Version 1.0 January 2009
%
% To compile to pdf, run:
% latex plos.template
% bibtex plos.template
% latex plos.template
% latex plos.template
% dvipdf plos.template
\documentclass[11pt]{article}
%\usepackage{natbib}
\usepackage{hyperref}
\usepackage{lineno}

% amsmath package, useful for mathematical formulas
\usepackage{amsmath}
% amssymb package, useful for mathematical symbols
\usepackage{amssymb}

% graphicx package, useful for including eps and pdf graphics
% include graphics with the command \includegraphics
\usepackage{graphicx}

% cite package, to clean up citations in the main text. Do not remove.
\usepackage{cite}

\usepackage{color} 

% Use doublespacing - comment out for single spacing
%\usepackage{setspace} 
%\doublespacing


% Text layout
\topmargin 0.0cm
\oddsidemargin 0.5cm
\evensidemargin 0.5cm
\textwidth 16cm 
\textheight 21cm
\linespread{1.2}
% Bold the 'Figure #' in the caption and separate it with a period
% Captions will be left justified
\usepackage[labelfont=bf,labelsep=period,justification=raggedright]{caption}

% Use the PLoS provided bibtex style
\bibliographystyle{plos2009}

% Remove brackets from numbering in List of References
\makeatletter
\renewcommand{\@biblabel}[1]{\quad#1.}
\makeatother


% Leave date blank
\date{}

\pagestyle{myheadings}
%% ** EDIT HERE **


%% ** EDIT HERE **
%% PLEASE INCLUDE ALL MACROS BELOW

%% END MACROS SECTION

\begin{document}

% Title must be 150 characters or less
\begin{flushleft}
{\Large
\textbf{Characterization of the transcriptome, nucleotide sequence polymorphism, and natural selection in the desert adapted mouse \textit{Peromyscus eremicus}}
}
% Insert Author names, affiliations and corresponding author email.
\\
Matthew D. MacManes$^{1}$, 
Michael B. Eisen $^{2}$, 
\\
$^{1}$ Department of Molecular, Cellular and Biomedical Sciences, University of New Hampshire, Durham NH, USA
\\
$^{2}$ HHMI and University of California, Berkeley, Berkeley, CA, USA
\\
$\ast$ E-mail: macmanes@gmail.com, @PeroMHC
\end{flushleft}

\linenumbers

% Please keep the abstract between 250 and 300 words
\section*{Abstract}



\section*{Introduction}

Deserts are widely considered one of Earth's harshest environments. Animals living in desert environments are forced to endure intense heat and drought, and in turn, species having evolved in these environments are likely ton have evolved specialised mechanisms that may enhance fitness. While living in deserts likely involves a large number of adaptive phenotypes, the ability to osmoregulate -- to maintain the proper water and electrolyte balance -- appears to be paramount \cite{Walsberg:2000uu}. Indeed, the maintenance of water balance in animals is one of the most important physiologic processes, and is critical to desert survival. Mammals are exquisitely sensitive to changes in osmolality, with slight derangement eliciting physiologic compromise.  When the loss of water exceeds dietary intake, dehydration - and in extreme cases, death - can occur. Understanding these mechanismsm will significantly enhance our understanding of the physiologic processes underlying osmoregulation in extreme environments, having implications for studies of human health, conservation, and climate change. \\

The genes and structures responsible for the maintinance of water and electrolyte balance are well characterized in model organisms such as mice \cite{Tatum:2009kp}, rats \cite{Romero:2007fs,Rojek:2006dg,NIELSEN:1995uq}, and humans \cite{Mobasheri:2007tt,Bedford:2003wa,NIELSEN:1999vg}. These studies, many of which have been enabled by newer sequencing technologies, serve as a foundation for studies of renal genomics in non-model organisms. In particular, because researchers have long been interested in desert adaptation, a number of studies have looked at the morphology or expression of single genes in the renal tissues of desert adapted rodents \textit{Phyllotis darwini} \cite{Gallardo:2005fm}, \textit{Psammomys obesus} \cite{Kaissling:1975uu}, and \textit{Perognathus penicillatus} \cite{Altschuler:1979du}. More recently, full renal transcriptomes have been generated for \textit{Dipodomys spectabilis} and \textit{Chaetodipus baileyi} \cite{Marra:2014de} as well as \textit{Abrothrix olivacea} \cite{Giorello:2014iv}. \\

These studies provide a rich context for the current and future work, aimed at developing a synthetic understanding of the the genetic and genomic underpinnings of desert adaptation in rodents. As a first step, we have sequenced, assembled, and characterized the transcriptome (using four tissue types - liver, kidney, testes, brain), of a desert adapted cricetid rodent endemic to the Southwest United States \cite{Veal:2001vp}, \textit{Peromyscus eremicus}. These animals have a lifespan typical of small mammals, and therefore an individual may live it's entire life without ever drinking water. These rodents have distinct advantage over other desert animals (e.g. \textit{Dipodomys}) in that they breed readily in captivity, which enables future laboratory studies of the phenotype of interest. In addition the focal species is positioned in a clade of well known animals (e.g. \textit{P. californicus}, \textit{P. maniculatus} and \textit{P. polionotus}) \cite{Bradley:2007jo} with growing genetic and genomic resources \cite{Shorter:2014kw,Panhuis:2011co,Shorter:2012ib} which together suggest that future comparative studies are possible. \\


While the elucidation of the mechanisms underlying adaptation to desert survival is beyond the scope of this manuscript, we aim here to lay the groundwork by characterizing the transcriptome from four distinct tissues (brain, liver, kidney, testes). These data will be included in current larger effort aimed at sequencing the entire genome. Further, via sequencing the renal tissue of a total of 15 additional animals, we characterize nucleotide polymorphism and genome wide patterns of natural selection. Together, these investigations will aid in our overarching goal -- to understand the genetic bases of adaptation in \textit{P. eremicus}.

\section*{Materials and Methods}

\subsection*{Animal Collection and Study Design}

To begin to understand how genes may underlie desert adaptation, I collected ~15 individuals from a single population \textit{P. eremicus} over a two year time period (2012-2013). These individuals were captured in live traps, then euthanized using isoflurane overdose and decapitation. Immediately post-mortem, the abdominal and pelvic organs were removed, cut in half (in the case of kidney), placed in RNAlater and flash frozen in liquid Nitrogen. Removal of the brain, with similar preservation techniques, followed that. Time from euthanasia to removal of all organs never exceeded five minutes. Samples were transferred to a -80C freezer at a later date. These procedures were approved by the University of California Berkeley Animal Care and Use Committee and follow guidelines established by the American Society of Mammalogy for the use of wild animals in research \cite{Sikes:2011dz}.  

\subsection*{RNA extraction and Sequencing}

Total RNA was extracted from each tissue using a TRIzol extraction (Invitrogen) following the manufacturers instructions. Because preparation of an RNA library suitable for sequencing is dependent on having high
quality, intact RNA, a small aliquot of each total RNA extract was
analyzed on a Bioanalyzer 2100 (Agilent, Palo Alto, CA, USA).
Following confirmation of sample quality, the reference sequencing libraries
were made using the TruSeq stranded RNA prep kit
(Illumina), while an unstranded TruSeq kit was used to construct the other sequencing libraries. A unique index was ligated to each sample to allow for multiplexed sequencing. Reference libraries (n=4 tissue types) were then pooled to
contain equimolar quantities of each individual library and
submitted for Illumina sequencing using two lanes of 150nt paired end sequencing using the rapid-mode of the HiSeq 2500 sequencer at The Hubbard Center for Genome Sciences (University of New Hampshire). The remaining 14 libraries were similarly multiplexed and sequenced in a mixture of 100nt paired and single end across two lanes of an Illumina HiSeq 2000 at the Vincent G. Coates Gnome Center (University of California, Berkeley).

\subsection*{Sequence Data Preprocessing and Assembly}

The raw sequence reads were error corrected using the software bless \cite{Heo:2014cb}, using kmer=25, based on the developers recommendations. The error corrected adapter and quality trimmed following recommendations from MacManes \cite{MacManes:2014io} and Mbandi \cite{Christoffels:2014bg}. Specifically, adapter sequence contamination was removed, and low quality nucleotides (defined as \textsc{Phred} \textless 2) were removed using the program Trimmomatic version 0.32  \cite{Lohse:2012fg}. Reads from each tissue were assembled using Trinity version released 17 July 2014 \cite{Haas:2013jq}. We used flags indicating the stranded nature of sequencing reads and set maximum allowable physical distance between read pairs to 999nt. The assembly was conducted on a linux workstation with 64 cores and 512Gb RAM. To filter the raw sequence assembly, I downloaded \textit{Mus musculus} cDNA and ncRNA datasets from Ensembl (\url{ftp://ftp.ensembl.org/pub/release-75/fasta/mus_musculus/}), and the \textit{Peromyscus maniculatus} reference transcriptome from NCBI (\url{ftp://ftp.ncbi.nlm.nih.gov/genomes/Peromyscus_maniculatus_bairdii/RNA/}). I used a relaxed blastN procedure (evalue set to 10$^-10$) to identify contigs in the \textit{P. eremicus} dataset that are likely biological in origin. I then concatenated the filtered assemblies from each tissue into a single file, reducing redundancy using the software cd-hit-est \cite{Li:2006hr} using default setting except that sequences were clustered based on 95\% sequence similarity. The resulting assembly file was characterized using the software package transrate (\url{https://github.com/Blahah/transrate}). \\

\subsection*{Assembled Sequence Annotation}

From the filtered assembly, I extracted putative coding sequences using Transdecoder version 4ul2014 (\url{http://transdecoder.sourceforge.net/}). These putative protein coding sequences were annotated using default settings of the blastx algorithm \cite{Camacho:2009fc} against the SwissProt database downloaded on 1 March 2014. Because transcriptome assemblies typically contain non-coding elements (e.g. ncRNA) in addition to protein coding sequence, we annotated the entire filtered dataset using the NCBI nt dataset, downloaded on 1 March 2014. Lastly, because rapidly evolving nucleotide sequences may evade detection by blast algorithms, we used HMMER3 \cite{Wheeler:2013gj} to search for conserved protein domains contained in the Pfam database \cite{Punta:2012ko}. \\

To identify sequences unique to each tissue type, I mapped sequence reads from each tissue type to the reference assembly using bwa-mem. We estimated expression individually for the four tissues. Interesting patterns of expression, including instances where expression was limited to a single tissue type were identified. \\ 

\subsection*{Population Genomics}

In addition to the reference individual sequenced at four different tissue types, we sequenced 15 other conspecific individuals from the same population, located in Palm Desert, California. Sequence data were mapped to the reference transcriptome using bwa-mem. The alignments were sorted and converted to BAM format, then passed to the program \textsc{Angsd}, which was used for calculating the folded site frequency spectrum (SFS) and Tajima's D \cite{Korneliussen:2013uz}. \\


\subsection*{Natural Selection}

To characterize natural selection on several genes related to water and ion homeostasis, we identified several of the transcripts identified as experiencing positive selection in a resent work on desert-adapter \textit{Dipodomys} rodents. The coding sequence corresponding to these genes, Solute Carrier family 2 member 9  (Slc2a9) and the Vitamin D3 receptor (Vdr), were extracted from the dataset, aligned using the software MACSE \cite{Ranwez:2011kj} to homologous sequences identified by the conditional reciprocal best blast procedure (CRBB, \cite{Aubry:2014en}) implemented in transrate. These alignments were inputted into the software codeABC version 1.6.0 \cite{Lopes:2013fd}.    


% Results and Discussion can be combined.
\section*{Results}

\subsection*{RNA extraction, Sequencing, Assembly, Mapping}

RNA was extracted from the hypothalamus, renal medulla, testes, or liver from each individual using sterile technique. TRIzol extraction resulted in a large amount of high quality (RIN $\geq$ 8) total RNA, which was used as input. Libraries were constructed as per the standard Illumina protocol, and ere sequenced as described above. The number of reads per library varied from 56 million strand-specific paired-end reads in Peer360 kidney, to 9 million single-end reads in Peer321. Adapter sequence contamination and low-quality nucleotides were eliminated, which resulted in a loss of \textless 2\% of reads. \\

Transcriptome assembly for each tissue type was accomplished using the program Trinity. The raw assembly for brain, liver, testes and kidney contained 185425, 222096, 180233, and 514091 assembled sequences respectively. This assembly was filtered using a blast procedure with resulted in a final dataset containing 68331 brain-specific transcripts, 71041 liver-specific transcripts, 67340 testes-specific transcripts, and 113050 kidney-specific transcripts. Mapping the error-corrected adapter/quality trimmed reads to these datasets resulted in mapping 94.98\% (87.01\% properly paired) of brain-derived reads to the brain transcriptome, 96.07\% (88.13\% properly paired) of liver-derived reads to the liver transcriptome, 96.81\% (85.10\% properly paired) of testes-derived reads to the testes transcriptome, and 91.87\% (83.77\% properly paired) of kidney-derived reads to the kidney transcriptome. Together, these statistics suggest that the tissue-specific transcriptomes are of extremely high quality.  \\

%\vspace{5mm}
%\textbf{\hypertarget{Figure 1}{Figure 1}} \\
\centerline{\includegraphics[width=20.0\baselineskip]{/Users/macmanes/Documents/pero_transcriptome/venn.jpeg}}
\begin{quote}
\small{Figure 1. The Venn Diagram.}
\end{quote}   

I then estimated gene expression on each of these tissue-specific datasets, which allowed me to understand expression patterns in the multiple tissues {\hypertarget{Figure 1}{Figure 1}}. After expression estimation, the filtered assemblies were concatenated together, and after removal of redundancy with cd-hit-est, 123,123 putative transcripts remained. From this filtered concatenated dataset, I extracted 71626 putative coding sequences (72Mb). Of these 71626 sequences, 38221 were complete exons (containing both start and stop codons), while other were either truncated at the 5-prime end (20239 sequences), 3-prime end (6445 sequences), or were internal (6721 sequencing having neither stop nor start codon). 






\subsection*{Subsection 2}

\section*{Discussion}

% You may title this section "Methods" or "Models". 
% "Models" is not a valid title for PLoS ONE authors. However, PLoS ONE
% authors may use "Analysis" 

% Do NOT remove this, even if you are not including acknowledgments
\section*{Acknowledgments}


%\section*{References}
% The bibtex filename
\bibliography{formatted.bib}

\section*{Figure Legends}
%\begin{figure}[!ht]
%\begin{center}
%%\includegraphics[width=4in]{figure_name.2.eps}
%\end{center}
%\caption{
%{\bf Bold the first sentence.}  Rest of figure 2  caption.  Caption 
%should be left justified, as specified by the options to the caption 
%package.
%}
%\label{Figure_label}
%\end{figure}


\section*{Tables}

\vspace{10mm}
\textbf{\hypertarget{Table 1}{Table 1}} \\
\begin{center}
\begin{tabular}{l|c }
\textsc{Dataset}& \textsc{Num. Raw Reads} \\
\hline 
\textsc{Peer360 Testes} & 32M PE  \\
\textsc{Peer360 Liver} & 53M PE   \\
\textsc{Peer360 Kidney} & 56M PE   \\
\textsc{Peer360 Brain}  & 23M PE \\
\textsc{Peer305}  & 19M PE  \\
\textsc{Peer308}  & 15M PE  \\
\textsc{Peer319}  & 14M PE  \\
\textsc{Peer321}  & 9M SE  \\
\textsc{Peer340}  & 16M PE  \\
\textsc{Peer352}  & 14M PE  \\
\textsc{Peer354}  & 9M SE  \\
\textsc{Peer359}  & 14M PE  \\
\textsc{Peer365}  & 16M PE \\
\textsc{Peer366}  & 16M PE  \\
\textsc{Peer368}  & 14M PE  \\
\textsc{Peer369}  & 14M PE  \\
\textsc{Peer372}  & 17M SE  \\
\textsc{Peer373}  & 23M SE  \\
\textsc{Peer380}  & 16M SE  \\
\textsc{Peer382}  & 14M SE  \\
\end{tabular}
\begin{quote}
\small{Table 1. The number of sequencing reads per sample}
\end{quote}


\end{center}  


\vspace{5mm}


\end{document}

